added notes on use of mamba in the installation documentation chapter (#2236)
ESMValGroup/ESMValTool: doc/sphinx/source/quickstart/installation.rst

here are too concise.
See `common installation issues`_ if you run into trouble.

.. _install_with_mamba:

Mamba installation
==================

In order to install the `conda <https://docs.conda.io>`_ package, you will need

> mamba pre-installed.

Author: this needs a bit of TLC: we should say one needs anaconda first (conda) then they need to install `mamba` in the `(base)` conda env with `conda install -c conda-forge mamba`

> For a minimal mamba installation (recommended) go to https://conda.io/miniconda.html.

Author: ```suggestion
For a minimal conda installation (recommended) go to https://conda.io/miniconda.html.
```

> It is recommended that you always use the latest version of mamba, as problems

Author: ```suggestion
It is recommended that you always use the latest version of `conda` and `mamba`, as problems
```

    bash Miniconda3-latest-Linux-x86_64.sh

and follow the instructions on your screen.
Immediately update mamba after installing it:

.. code-block:: bash

>     mamba update --name base mamba

Author: no, these should not change, conda needs to be updated, then installing mamba from conda-forge insures that mamba is at the latest version; if they need to update conda then they can do `conda update -c conda-forge mamba`

.. code-block:: bash

    mamba create --name esmvaltool -c conda-forge esmvaltool python=3.9

Here ``mamba`` is the executable calling the mamba package manager to install
``esmvaltool`` and the ``-c`` flag specifies the conda software channels in which the
``esmvaltool`` package and its dependencies can be found.
The reason why we are also specifying ``python=3.9`` is that it will make it
easier for mamba to find a working combination of all required packages, see

> `Mamba fails to solve the environment`_ in `common installation issues`_ for an

Author: we should have a look at those common issues, they may differ for mamba

`conda environment <https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#managing-environments>`_
and install ESMValTool into it with a single command.

.. code-block:: bash

    conda activate esmvaltool

Of course it is also possible to choose a different name than ``esmvaltool`` for the environment.

`mamba <https://github.com/mamba-org/mamba>`_ is a fast alternative to ``conda`` and an installation using ``mamba`` is possible as well, and may prove to be faster. To use ``mamba``, you will have to install it first, in the ``(base)`` environment:

.. code-block:: bash

    conda install -c conda-forge mamba

then repeat the same steps as above but using ``mamba`` instead of ``conda``:

.. code-block:: bash

    mamba create --name esmvaltool
    mamba activate esmvaltool

>     mamba install -c conda-forge -c esmvalgroup esmvaltool

Author: ```suggestion
    mamba install -c conda-forge esmvaltool
```

or in one single command:

.. code-block:: bash

>     mamba create --name esmvaltool -c esmvalgroup -c conda-forge esmvaltool

Author: ```suggestion
    mamba create --name esmvaltool -c conda-forge esmvaltool
```

The ``esmvaltool-python`` diagnostics can be installed as follows:

> First, ensure mamba is pre-installed (see `Mamba installation`_ for more details).

Author: ```suggestion
First, ensure both conda and mamba are pre-installed (see `Mamba installation`_ for more details).
```

It is also possible to install ESMValTool from `PyPI <https://pypi.org/project/ESMValTool/>`_.
However, this requires first installing dependencies that are not available on PyPI in some other way.
By far the easiest way to install these dependencies is to use mamba_.

> For a minimal mamba installation (recommended) go to https://conda.io/miniconda.html.

Author: here too: install anaconda/miniconda, then mamba from conda-forge in (base)

Prerequisites
-------------

It is recommended to use mamba to manage ESMValTool dependencies.

> For a minimal mamba installation go to https://conda.io/miniconda.html. To

Author: here too: miniconda first, then mamba from conda-forge

simplify the installation process, an environment definition file is provided
in the repository (``environment.yml`` in the root folder).

.. attention::
    Some systems provide a preinstalled version of conda (e.g., via the module environment).
    However, several users reported problems when installing NCL with such versions. It is
    therefore preferable to use a local, fully user-controlled mamba installation.
    Using an older version of mamba can also be a source of problems, so if you have mamba

>     installed already, make sure it is up to date by running ``mamba update -n base mamba``.

Author: no no, conda needs to be updated, then mamba

.. note::
    Note that during the installation, mamba will ask you
    if you want the installation to be automatically sourced from your

>     ``.bashrc`` or ``.bash-profile`` files; if you answered yes, then mamba

Author: does it? I've only done the miniconda install
